pkg/volume/util.go

```diff
@@ -47,6 +50,21 @@ func GetAccessModesAsString(modes []api.PersistentVolumeAccessMode) string {
 	return modesAsString
 }
 
+func GetAccessModesFromString(modes string) []api.PersistentVolumeAccessMode {
```

Comment:
Seems like these are part of the API, not volumes.  Why not move these to api/helpers.go?